lib/orwell/web/controllers/post_controller.ex

```diff
@@ -82,4 +78,10 @@ defmodule Orwell.Web.PostController do
 
     %{"id" => current_max_id + 1}
   end
+
+  defp create_post(post, config) do
+    filename = Orwell.GitHub.Blog.to_filename(post.id, post.title)
+    {:ok, _url} = Orwell.GitHub.commit(filename, post.body, config)
```

Comment:
Note that this is only using the `post.body`. The actual file content isn't formatted correctly as you can see in subvisual/blog#23